docs/unittest.md

```diff
@@ -0,0 +1,102 @@
+# Write and run unittest
+
+## How to run unittest
+
+[OceanBase](https://github.com/oceanbase/oceanbase) has two unittest directories.
+
+- `unittest` . This is the main unittest cases, and it tests the code in the `src` directory.
+
+- `deps/oblib/unittest`. Cases for oblib.
+
+First, you should build `unittest`. You should enter the `unittest` directory in the build directory and build explicitly. When you build the oceanbase project, it does't build the unittest in default. For example:
+
+```bash
+bash build.sh --init --make # init and build a debug mode project
+cd build_debug/unittest  # or cd build_debug/deps/oblib/unittest
+make # build unittest
+```
+
+Then you can execute the script file `run_tests.sh` to run all test cases.
+
+## How to build and run a single unittest
+
+You can also build and test a single unittest case. You can enter the `build_debug` directory, and execute `make case-name` to build the specific case and run the binary file built. For example:
+
+```bash
+cd build_debug
+# **NOTE**: don't enter the unittest directory
+make test_chunk_row_store
```

Comment:
may be you can add `-j4` here? By doing this, let others know that parallelism can be added here; otherwise, the compilation will be relatively slow.